nucleus/common/common-util/src/main/java/com/sun/enterprise/util/io/FileUtils.java

```diff
@@ -1336,7 +1349,13 @@ public static String resourceToString(String resourceName) {
                     is.close();
                 }
                 catch (Exception ex) {
-                    // ignore...
+                	_utillogger.log(Level.SEVERE, CULoggerInfo.exceptionIO, ex);
+                }finally{
+                	   try {
+						is.close();
```

Comment:
this is a very weird construct. Why there is is.close() on 3 places? 1 should be enough:)
Why no to add finally to the outer try/catch block and close it there?